Script/Packages/Atomic/Container.json

```diff
@@ -1,5 +1,10 @@
 {
 	"name" : "Container",
 	"sources" : ["Source/Atomic/Container"],
-	"classes" : ["RefCounted"]
+	"classes": [ "RefCounted" ],
+	"overloads" : {
+		"RefCounted" : {
+			"RefCounted" : []
+		}
```

Comment:
Do we need access to the RefCounted constructor?  These should never be instantiated, perhaps missing something here